.github/workflows/terraform_security_scan.yml

```
@@ -0,0 +1,30 @@
+name: "Terraform security scan"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "terragrunt/**"
+      - ".github/workflows/terraform-security-scan.yml"
+  pull_request:
+    paths:
+      - "terragrunt/**"
+      - ".github/workflows/terraform-security-scan.yml"
```

You'll just need to tweak the filenames to use `_`:
```suggestion
      - ".github/workflows/terraform_security_scan.yml"
  pull_request:
    paths:
      - "terragrunt/**"
      - ".github/workflows/terraform_security_scan.yml"
```

---

Aah! Thank you - should have caught this.